keps/sig-storage/2268-non-graceful-shutdown/README.md

```diff
@@ -286,15 +324,25 @@ _This section must be completed when targeting beta graduation to a release._
   Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
   checking if there are objects with field X set) may be a last resort. Avoid
   logs or events for this purpose.
+  An operator, the person who operates the cluster, can check if the
+  `NodeOutOfServiceVolumeDetach` feature gate is enabled and if there is an
+  `out-of-service` taint on the shutdown node.
+  The usage of this feature requires the manual step of applying a taint
+  so the operator should be the one applying it.
 
 * **What are the SLIs (Service Level Indicators) an operator can use to determine
 the health of the service?**
   - [ ] Metrics
-    - Metric name:
+    - Metric name: We can add new metrics deleting_pods_total, deleting_pods_error_total
+      in Pod GC Controller.
+      For Attach Detach Controller, there's already a metric:
+      attachdetach_controller_forced_detaches.
```

Comment:
Do we want to distinguish between 6 minute force detach vs force detach by taint?

Reply:
Yeah, it is probably good to distinguish that.  Do we need to keep the original metric name "attachdetach_controller_forced_detaches" and add a new one for detach by taint? Or have two taints "attachdetach_controller_forced_detaches_timeout" and "attachdetach_controller_forced_detaches_taint"?

Reply:
I kept the original metric and added a new for taint.

Reply:
The current metric is alpha stage, so we can potentially make changes to it. What if we add a new label to the same metric?